pkg/controller/virtualmachine/virtualmachine_controller.go

```diff
@@ -79,32 +80,37 @@ type ReconcilePolicy struct {
 
 // Reconcile reads that state of the cluster for a virtual machine object and makes changes based on the state
 func (r *ReconcilePolicy) Reconcile(request reconcile.Request) (reconcile.Result, error) {
-	log.V(1).Info("got a virtual machine event in the controller",
-		"virtualMachineName", request.Name,
-		"virtualMachineNamespace", request.Namespace)
+	log = logf.Log.WithName("VirtualMachine Controller").WithValues("virtualMachineName", request.Name, "virtualMachineNamespace", request.Namespace)
```

Comment:
Use different variable, you don't want to replace global log here.

Reply:
but it's [set](https://github.com/k8snetworkplumbingwg/kubemacpool/blob/3fe1379b3c1744976ee89258dde900c964971294/pkg/controller/virtualmachine/virtualmachine_controller.go#L38) in the start of the module. Should I move it from there?

Reply:
You can use the global log to generate a new one so we have the WithName just do the following
 ```golang
logger := log.WithName("Reconcile").WithValues("virtualMachineName", request.Name, "virtualMachineNamespace", request.Namespace)
```
And use logger at that function, I think is bot "names" will appear "VirtualMachine Controller" and "Reconcile"

Reply:
ack, (cool!)